### ISSUE TITLE
Cursor keeps jumping to top of document

### ISSUE THREAD

**Issue by fedi-will** [ryanmcalister/unotes#145]
Most of the time when I open a .md file with unotes the cursor jumps to line 1, col 1 after about 2 seconds. 
That means, whereever I click in the document to set the cursor to edit a text passage, the cursor jumps to top and I am putting text into the first line. 

This makes using this extension impossible. 

When this happens, the "working" indicator constantly appears in the unotes left side folder view. It looks as if I would constantly click the "Refresh Tree" button. 

This is the same thing as the user review from "keef" on the vscode marketplace, where he says: 
```
would love to use this - love the idea...
But my cursor keeps jumping to the top of the document - I am guessing on the auto save..
This unfortunately makes it unusable :(
```

**Comment by fedi-will** [comment]
Ok, it's not reproducible. I played around a bit and now it works. 
This happened on files that existed before I installed this extension, but now it is ok on these files, too. 

FYI: 
VSCode version: 1.73.0
unotes version: 1.4.4

Could it be that under certain circumstances the extension keeps refreshing and thus keeps reloading my opened document??

**Comment by kiyoka** [comment]
I fixed it in #141 .
It is now fixed in Unotes Version 1.5, please try Unotes Version 1.5.

**Comment by fedi-will** [comment]
Looks like this resolves the problem. After short test on my Win PC and laptop. 
Thanks
